provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/location/LocationController.java

```diff
@@ -1,3 +1,183 @@
 package gov.va.api.health.providerdirectory.service.controller.location;
 
-public class LocationController {}
+import gov.va.api.health.providerdirectory.service.CareSitesResponse;
+import gov.va.api.health.providerdirectory.service.CountParameter;
+import gov.va.api.health.providerdirectory.service.LocationWrapper;
+import gov.va.api.health.providerdirectory.service.client.PpmsClient;
+import gov.va.api.health.providerdirectory.service.controller.Bundler;
+import gov.va.api.health.providerdirectory.service.controller.Bundler.BundleContext;
+import gov.va.api.health.providerdirectory.service.controller.PageLinks.LinkConfig;
+import gov.va.api.health.providerdirectory.service.controller.Parameters;
+import gov.va.api.health.providerdirectory.service.controller.Validator;
+import gov.va.api.health.stu3.api.resources.Location;
+import gov.va.api.health.stu3.api.resources.OperationOutcome;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import javax.validation.constraints.Min;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Request Mappings for the Location Profile, see
+ * http://www.fhir.org/guides/argonaut/pd/StructureDefinition-argo-location.html for implementation
+ * details.
+ */
+@SuppressWarnings("WeakerAccess")
+@RestController
+@RequestMapping(
+  value = {"/api/Location"},
+  produces = {"application/json", "application/json+fhir", "application/fhir+json"}
+)
+public class LocationController {
+
+  private Transformer transformer;
+
+  private PpmsClient client;
+
+  private Bundler bundler;
+
+  /** Controller setup. */
+  public LocationController(
+      @Autowired LocationController.Transformer transformer,
+      @Autowired Bundler bundler,
+      @Autowired PpmsClient client) {
+    this.transformer = transformer;
+    this.bundler = bundler;
+    this.client = client;
+  }
+
+  private Location.Bundle bundle(MultiValueMap<String, String> parameters, int page, int count) {
+    LocationWrapper locationWrapper = LocationWrapper.builder().build();
+    CareSitesResponse careSitesResponse;
+    List<LocationWrapper> paged = new ArrayList<>();
+    int recordCount = 1;
```

Comment:
Call this `totalRecords` to match how it is used below. Otherwise it's not clear if this is all the records or the paged subset.